### PR TITLE
Bug/aapd 82 unable to see added users

### PR DESCRIPTION
**Pull request by suityou01**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-82

## Description of change

Added id param to routes on manage-appeals/your-email-address route

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Comment by suityou01** [comment]
> I don't understand this I'm afraid Why would we need to find out the user's email via this login page if we already have the user's id in the url?
> 
> Surely the your email page should not use an id in the url and then on post it should look the user up by email and redirect them onto the enter code with their id in that url?

You and me both. If you look at the controller it expects an id from the url path.

File: packages/forms-web-app/src/controllers/common/your-email-address.js [hunk 1]
```diff
@@ -32,6 +29,8 @@ const postYourEmailAddress = (views) => {
 		}
 
 		const { email } = body['email-address'];
+		const id = await getUserByEmail(email);
```
Comment (bryn-pins) [inline packages/forms-web-app/src/controllers/common/your-email-address.js:32]: Believe this returns the actual user so will need to use the _id prop from this method call

Realise there's no jsdocs on the api wrapper and no openapi specs
Was looking into adding some automated openapi specs and will try and find time to achieve that soon

**Review by bryn-pins — CHANGES_REQUESTED** [review]

**Review by bryn-pins — APPROVED** [review]

**Review by bryn-pins — DISMISSED** [review]

**Comment by chrishawldev** [comment]
We don't actually need the id in the url, I think this is just from the Heroku template so feel free to just remove the id from the controller action if we don't need it. I'd rather we did that anyway so Google analytics isn't scraping all of our user id's

**Review by bryn-pins — APPROVED** [review]

File: packages/forms-web-app/src/controllers/common/your-email-address.js [hunk 2]
```diff
@@ -30,6 +28,9 @@ const postYourEmailAddress = (views) => {
 		}
 
 		const { email } = body['email-address'];
+		const user = await getUserByEmail(email);
+		const id = user._id;
```
Comment (chrishawldev) [inline packages/forms-web-app/src/controllers/common/your-email-address.js:32]: Nice one

**Comment by bryn-pins** [comment]
Closing as fixed in https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/2980

**Pull request closed without merging**